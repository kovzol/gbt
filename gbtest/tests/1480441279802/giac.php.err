// Using locale /usr/share/locale/
// C
// /usr/share/locale/
// giac
// UTF-8
// Maximum number of parallel threads 8
Help file /usr/share/giac/doc/local/aide_cas not found
Added 0 synonyms

// Groebner basis computation time 0.003634 Memory 0.025324M
// Time 0
0.04user 0.00system 0:00.05elapsed 96%CPU (0avgtext+0avgdata 25324maxresident)k
0inputs+16outputs (0major+1619minor)pagefaults 0swaps
