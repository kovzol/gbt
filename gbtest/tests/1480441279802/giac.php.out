Welcome to giac readline interface
(c) 2001,2016 B. Parisse & others
Homepage http://www-fourier.ujf-grenoble.fr/~parisse/giac.html
Released under the GPL license 3.0 or above
See http://www.gnu.org for license details
May contain BSD licensed software parts (lapack, atlas, tinymt)
-------------------------------------------------
Press CTRL and D simultaneously to finish session
Type ?commandname for help
0>> gbasis([(x1),(x2),((((x2)-(1))*(x3))-((x1)*((x4)-(1)))),(x6),((0)+(((x5)^<2)-(1))*(x3))-((x1)*((x4)-(1)))),(x6),((0)+(((x5)^2                         )+(((x6)^2)+(((((x1)^2)+(<(1)))),(x6),((0)+(((x5)^2)+(((x6)^2)+(((((x1)^2)+((                         x2)^2))-(((-(x3))*(-(x3))<+(((x6)^2)+(((((x1)^2)+((x2)^2))-(((-(x3))*(-(x3)))                         +(((1)-(x4))*((1)-(x4))))<2)^2))-(((-(x3))*(-(x3)))+(((1)-(x4))*((1)-(x4)))))                         +(((0)*((x5)*(x6)))+((((2<(((1)-(x4))*((1)-(x4)))))+(((0)*((x5)*(x6)))+((((2)                         *(-(x1)))*(x5))+(((2)*(-(<(((0)*((x5)*(x6)))+((((2)*(-(x1)))*(x5))+(((2)*(-(x                         2)))*(x6)))))))),((((x6)-<(-(x1)))*(x5))+(((2)*(-(x2)))*(x6)))))))),((((x6)-(                         x4))*((x)-(x3)))-(((x5)-(<)))*(x6)))))))),((((x6)-(x4))*((x)-(x3)))-(((x5)-(x                         3))*((y)-(x4)))),-2*x4^2*<4))*((x)-(x3)))-(((x5)-(x3))*((y)-(x4)))),-2*x4^2*x                         2+2*x4^2-2*x4*x3*x1+2*x4*<))*((y)-(x4)))),-2*x4^2*x2+2*x4^2-2*x4*x3*x1+2*x4*x                         2*y+2*x4*x2-2*x4*y-2*x4+2<+2*x4^2-2*x4*x3*x1+2*x4*x2*y+2*x4*x2-2*x4*y-2*x4+2*                         x3*x1*y+2*x2*x1*x5-2*x2*x<*y+2*x4*x2-2*x4*y-2*x4+2*x3*x1*y+2*x2*x1*x5-2*x2*x1                         *x-2*x2*x5^2+2*x2*x5*x-2*<3*x1*y+2*x2*x1*x5-2*x2*x1*x-2*x2*x5^2+2*x2*x5*x-2*x                         2*y-2*x1^2*x6+2*x1^2*y+2*<x-2*x2*x5^2+2*x2*x5*x-2*x2*y-2*x1^2*x6+2*x1^2*y+2*x                         1*x5*x6-2*x1*x5*y-2*x1*x5<*y-2*x1^2*x6+2*x1^2*y+2*x1*x5*x6-2*x1*x5*y-2*x1*x5+                         2*x1*x+2*x5^2-2*x5*x+2*y]<*x5*x6-2*x1*x5*y-2*x1*x5+2*x1*x+2*x5^2-2*x5*x+2*y],                         [x6,x5,x4,x3,x2,x1,x,y],r<*x1*x+2*x5^2-2*x5*x+2*y],[x6,x5,x4,x3,x2,x1,x,y],re                         vlex)
[x^4-x^2*y^2+4*x4*y^3-18*x^2*y-20*x4*y^2-4*y^3-32*x5*x+19*x^2+28*x4*y+20*y^2-12*x4-28*y+12,x4*x*y^2+x5*y^3+x^3+6*x4*x*y+x5*y^2+x4*x-5*x5*y-4*x*y+3*x5-4*x,x5*x^2+3*x4*x*y+2*x5*y^2+x4*x-4*x5*y-x*y+2*x5-3*x,x4*x^2-x4*y^2-4*x5*x+3*x^2+2*x4*y+y^2-x4-2*y+1,x5*x*y+x4*y^2+3*x5*x-2*x^2-2*x4*y-y^2+x4+2*y-1,2*x5^2-x5*x-x4*y+x4+y-1,x5*x4-x4*x-x5*y,2*x4^2-x5*x-x4*y-3*x4+y+1,x6,x3,x2,x1]
1>> 